ci/rpm7/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:centos7
+
+ADD mongodb.repo /etc/yum.repos.d/
+ADD build.sh /opt/bin/
+ADD build-dep.sh /opt/bin/
+ADD makefile /opt/archive/
+
+RUN ln -s /opt/bin/build.sh /usr/local/bin/build \
```

Comment:
I undertand we don't need to create the symlink if we use `build.sh` instead of `build` in .travis.yml, isn't it?

Reply:
Ok. It doesn't hurt. NTC